build.gradle

```diff
@@ -38,7 +41,7 @@ plugins {
 }
 
 allprojects {
-    version "$confidential_id_release_version"
+    version "$confidential_id_release_version$confidential_id_versionSuffix"
```

Comment:
Shouldn't we have some delimeter between the version and the suffix, e.g. a dash or dot ?